### ISSUE TITLE
No capable of entering in Safe mode.

### ISSUE THREAD

**Issue by kimmansur** [checkra1n/BugTracker#1651]
**Tell us about your setup:**
1. What iDevice are you using? iPhone 7 Plus
2. On what version of iOS is it? 13.6.1
3. What version of checkra1n are you using? 0.10.2
4. What is your host system (OS version? Hackintosh? VM? etc.)? Mac OS Catalina
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
I'm using a Macbook Pro 2019, so all my ports are USB-C, but I was able to jailbreak using my USB hub from Baseus for a week or two, no errors before. It was all perfect.

**What are the steps to reproduce the issue?**
1. After Cydia Installer update yesterday, my iPhone rebooted and i wasn't able to re-jailbreak.
2. Tried different cables, but as i said all my cables are good.
3. Tried to use only safe mode box on.
4. Tried to use safe mode and verbose boot. 
5. Tried to safe mode, holding up volume after apple logo shows up.
...

**What do you expect, and what is happening instead?**
Some help to put this device in safe mode for adjustments on the tweaks updates.

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
It gives error -20
The full error is:

Timed out waiting for bootstrap upload (Likely caused by an incompatible tweak, to enable Safe Mode check the option on main screen) (Error code: -20)

**Any other info, error logs, screenshots, ...?**

**Comment by Siguza** [comment]
Duplicate of #1613.

**Comment by kimmansur** [comment]
> Duplicate of #1613.

Hello,
I believe that my issue is a little different. But maybe you can help me. can I remove the jailbreak, without check rain installed? Because my jailbreak process can't be finish, also I can't but it in safe mode.

**Comment by Siguza** [comment]
I mean, you can always do an IPSW restore...

**Comment by kimmansur** [comment]
> I mean, you can always do an IPSW restore...

No other way? 🙄🙏🏻

**Comment by kimmansur** [comment]
I was able to solve using imazing and restoring the iOS. After that I was able to rejailbreak. Lost all tweaks but no needed to resync all data.

**Comment by asrgaohifd** [comment]
> 
> 
> I was able to solve using imazing and restoring the iOS. After that I was able to rejailbreak. Lost all tweaks but no needed to resync all data.

How?

**Comment by kimmansur** [comment]
> > 
> 
> > 
> 
> > I was able to solve using imazing and restoring the iOS. After that I was able to rejailbreak. Lost all tweaks but no needed to resync all data.
> 
> 
> 
> How?

Sorry my delay.
I just used this option in the image below, it is in Portuguese, but in English would be ****reinstall iOS**** after this process I was able to redo the jailbreak process, on an iPhone 7plus. 
I used this process twice already.

 ![image](https://user-images.githubusercontent.com/16175701/93928671-adfef180-fce8-11ea-81a2-87eb2c9c3f90.jpeg)
